app/test_main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_can_access_google_page(
+        mocked_has_internet_connection: mock.MagicMock,
+        mocked_valid_google_url: mock.MagicMock
+) -> None:
+    can_access_google_page("https://mate.academy/")
+    mocked_has_internet_connection.assert_called_once()
+    mocked_valid_google_url.assert_called_once_with("https://mate.academy/")
```

Comment:
You are hardcoding the URL `https://mate.academy/` in your test. This is not a good practice as it makes your test less flexible and harder to maintain. It's better to define such constants at the beginning of your test or even better, to use pytest fixtures or parameters. This way, if the URL changes in the future, you only need to update it in one place.